.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+
+node_js:
+  - "6"
+
+cache:
+  directories:
+    - node_modules
+
+script:
+  - npm run test
```

Comment:
Unless this is a built in magic script target I don't see "test" defined anywhere in here.